frontend/apps/app/app/(app)/app/organizations/[organizationId]/projects/page.tsx

```diff
@@ -0,0 +1,48 @@
+import type { PageProps } from '@/app/types'
+import { ProjectsPage } from '@/features/projects/pages'
+import { migrationFlag } from '@/libs'
+import { createClient } from '@/libs/db/server'
+import { notFound } from 'next/navigation'
+import React from 'react'
+import * as v from 'valibot'
+
+const paramsSchema = v.object({
+  organizationId: v.string(),
+})
+
+export default async function Page({ params }: PageProps) {
+  const parsedParams = v.safeParse(paramsSchema, await params)
+  if (!parsedParams.success) return notFound()
+  const { organizationId } = parsedParams.output
+
+  const migrationEnabled = await migrationFlag()
+
+  if (!migrationEnabled) {
+    notFound()
+  }
```

Comment:
It seems unnecessary because it is already checked in middleware.ts.

https://github.com/liam-hq/liam/blob/f9c49fb445b2d7844d95636c62594e4fd9461bc3/frontend/apps/app/middleware.ts#L13